### PR TITLE
#564: Added cheat for charging a spineshark

### PR DESCRIPTION
**Pull request by CPColin**
Shout-out to that leftover debug routine in the ROM that does the same thing, for showing me what RAM addresses to set!

### CODE REVIEW

**Review by MattEqualsCoder — APPROVED** [review]

File: src/TrackerCouncil.Smz3.Tracking/VoiceCommands/CheatsModule.cs [hunk 1]
```diff
@@ -207,6 +207,15 @@ private GrammarBuilder SetupCrystalFlashRule()
             .OneOf("setup crystal flash requirements", "ready a crystal flash");
     }
 
+    [SupportedOSPlatform("windows")]
+    private GrammarBuilder ChargeShinesparkRule()
+    {
+        return new GrammarBuilder()
+            .Append("Hey tracker,")
+            .Optional("please", "would you kindly")
+            .OneOf("charge a shine spark", "ready a spine shark");
```
Comment (Vivelin) [inline src/TrackerCouncil.Smz3.Tracking/VoiceCommands/CheatsModule.cs:216]: After the fact: maybe we should add the lines that Betus tried? "Give me (a) shine spark", "Activate shine spark" and "Enable shine spark". Maybe also "turn on shine spark".

Comment (CPColin) [reply src/TrackerCouncil.Smz3.Tracking/VoiceCommands/CheatsModule.cs:216]: Good idea. Let me put something together.

Comment (MattEqualsCoder) [reply src/TrackerCouncil.Smz3.Tracking/VoiceCommands/CheatsModule.cs:216]: Based on the recent discussion about uses, maybe you could add something like "I don't want to have to IBJ". lol

Comment (CPColin) [reply src/TrackerCouncil.Smz3.Tracking/VoiceCommands/CheatsModule.cs:216]: Chaos mode where Tracker is listening for a bunch of phrases that don't start with "Hey Tracker"

**Pull request merged**